win32/src/win32apimodule.cpp

```diff
@@ -5390,6 +5390,48 @@ static PyObject * PyLoadString(PyObject *self, PyObject *args)
 	return rc;
 }
 
+// @pymethod PyHANDLE|win32api|LoadResource|Finds a resource in a PE file. Returns a HRSRC(HANDLE) to the resource if found, otherwise returns None.
+static PyObject * PyFindResource(PyObject *self, PyObject *args) {
+    
+    PyObject *ret;
+    
+    PyObject *obhModule;
+    PyObject *obName;
+    PyObject *obType;
+    
+    HMODULE hModule;
+    LPTSTR lpName = NULL;
+    LPTSTR lpType = NULL;
+    
+    if ( !PyArg_ParseTuple(args, "OOO|H:FindResource",
```

Comment:
This call's format string expects a 4th int param to be specified with a default value already set.

Reply:
I fixed the call. As I said, I am rusty on CPython and I based my code on existing examples I saw in the existing code without delving too deep into them